spec/features/admin/merchants/index_spec.rb

```diff
@@ -4,11 +4,11 @@
   describe "As an admin" do
     describe "I visit the admin merchants index (/admin/merchants)" do
       before :each do
-        @merchant_1 = Merchant.create!(name: "Johns Tools")
-        @merchant_2 = Merchant.create!(name: "Hannas Hammocks")
-        @merchant_3 = Merchant.create!(name: "Pretty Plumbing", active_status: :disabled)
-        @merchant_4 = Merchant.create!(name: "Jenna's Jewlery")
-        @merchant_5 = Merchant.create!(name: "Sassy Soap", active_status: :disabled)
+        @merchant_1 = Merchant.create!(name: "Johns Tools", active_status: :enabled)
+        @merchant_2 = Merchant.create!(name: "Hannas Hammocks", active_status: :enabled)
+        @merchant_3 = Merchant.create!(name: "Pretty Plumbing")
+        @merchant_4 = Merchant.create!(name: "Jenna's Jewlery", active_status: :enabled)
+        @merchant_5 = Merchant.create!(name: "Sassy Soap")
```

Comment:
Thanks for correcting these when we realized that the enum was created with the wrong assignment saved us some headache on fixing these tests later.